### PR TITLE
Make it way more easier to understand.

### PR DESCRIPTION
**Pull request by bruvv**

### CODE REVIEW

File: ui/darksky/README.md [hunk 1]
```diff
@@ -6,11 +6,17 @@ the card's capabilities through configuration flags.
 
 **Installation**
 ------------------------------
-1. Add **dark-sky-weather-card.js** to your **www/custom_ui/** directory.  If you don't have this directory (this is your first
-custom card), you will need to create it.
+1. Add ```dark-sky-weather-card.js``` to your ```<config-dir>*www/custom_ui/``` directory.  If you don't have this directory (this is your first custom card), you will need to create it.
```
Comment (iammexx) [inline ui/darksky/README.md]:
~~~~
to your ```<config-dir>*www/custom_ui/```  
~~~~
missing forward slash

File: ui/darksky/README.md [hunk 2]
```diff
@@ -6,11 +6,17 @@ the card's capabilities through configuration flags.
...
 
-2. Download the amcharts icons from https://www.amcharts.com/dl/svg-weather-icons/ and put them in **www/icons/weather_icons**.  Create 
-the directories if necessary. You should end up with an **animated** directory and a **static** directory inside the **weather_icons** directory.
+2. Download the amcharts icons from https://www.amcharts.com/dl/svg-weather-icons/ and put them in ```<config-dir>www/icons/weather_icons```.  Create the directories if necessary.
```
Comment (iammexx) [inline ui/darksky/README.md]:
~~~~
and put them in ```<config-dir>www/icons/weather_icons```. 
~~~~
missing forward slash

File: ui/darksky/README.md [hunk 3]
```diff
@@ -6,11 +6,17 @@ the card's capabilities through configuration flags.
...
+
+You should end up with the following folders:
+
+```<config-dir>/www/custom/ui/dark-sky-weather-card.js```
```
Comment (iammexx) [inline ui/darksky/README.md]:
~~~~
You should end up with the following folders:

 ```<config-dir>/www/custom/ui/dark-sky-weather-card.js```
~~~~
 Not a directory ... and the path is incorrect   .../custom_ui/   not .../custom/ui/

File: ui/darksky/README.md [hunk 4]
```diff
@@ -57,7 +63,7 @@ The next two steps are completed differently based on the version of HA you are
 - Pre 0.84 or if using yaml mode in 0.84 or above : Add to your ui-lovelace.yaml file.
 - Using storage mode in 0.84 or above use the "Raw Config Editor" to add the reference and definition to the config.
 
-2. Add the card reference at the top of the configuration ***Note the use of type: module and not js
+2. Add the card reference at the top of the configuration ***Note the use of type: module and not js **Note2: /local/ points to the /config/www/ dir.
```
Comment (iammexx) [inline ui/darksky/README.md]:
~~~~
 **Note2: /local/ points to the /config/www/ dir.
~~~~
should follow same convention as your other edits. eg:
~~~~
 **Note2: /local/ points to the <config-dir>/www/ dir.
~~~~

**Review by iammexx — CHANGES_REQUESTED** [review]
Thanks for submitting the changes...  I agree that it makes the docs easier to understand.  A couple of minor updates are needed before merging.

**Comment by d1slact0r** [comment]
Thanks for the review, fixed!

**Review by bruvv — COMMENTED** [review]
fixed

**Comment by bruvv** [comment]
Fixed *

**Pull request merged**